sdk/README.md

```diff
@@ -20,6 +20,8 @@ Here we update the `Compiler` of the KFP SDK to generate `Tekton` YAML for a bas
  - Tekton: [`0.11.0`](https://github.com/tektoncd/pipeline/releases/tag/v0.11.0-rc1)
  - Tekton CLI: [`0.8.0`](https://github.com/tektoncd/cli/releases/tag/v0.8.0)
 
+In order to use parameter passing from task outputs into condition parameters Tekton must be built from master. 
```

Comment:
master will continue to change, maybe also add the commit and date or PR that brought the feature into Tekton and the expected release that has it